__tests__/Unit/pages/Progress/ProgressUpdatesPage.test.tsx

```diff
@@ -16,9 +16,11 @@ describe('Progress page test', function () {
         );
         const textBoxes = screen.getAllByRole('textbox');
         expect(textBoxes).toHaveLength(3);
-        const button = screen.getByRole('button') as HTMLButtonElement;
-        expect(button).toBeInTheDocument();
-        expect(button.type).toBe('submit');
+        const buttons = screen.getAllByRole('button') as Array<HTMLButtonElement>;
```

Comment:
Hey sai kiran @WYGIN  why you are changing progressUpdatePage unit test?
Can you please give some context??

Reply:
Thank you Sai Kiran for giving very good explanation